pkg/kclient/volumes.go

```diff
@@ -13,6 +15,9 @@ import (
 const (
 	PersistentVolumeClaimKind       = "PersistentVolumeClaim"
 	PersistentVolumeClaimAPIVersion = "v1"
+
+	// The length of the string to be generated for names of resources
+	nameLength = 5
```

Comment:
I'm assuming we'd want to use this constant through kclient so makes sense to move it to kclient.go?

Reply:
updated